src/tmx/TmxUtils/test/J2735MessageTest.cpp

```diff
@@ -539,6 +539,124 @@ TEST_F(J2735MessageTest, EncodeBasicSafetyMessage)
 	free(message);
 	free(frame_msg.get_j2735_data().get());
 	ASSERT_EQ(20,  bsmEncodeMessage.get_msgId());
+	//Decode the encoded BSM
+	auto bsm_ptr = bsmEncodeMessage.decode_j2735_message().get_j2735_data();
+}
+
+
+
+TEST_F(J2735MessageTest, EncodeBasicSafetyMessag_PartII)
```

Comment:
Typo, should be `EncodeBasicSafetyMessag_PartII`

Reply:
https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/438/commits/1133e936f7532927db36fb6d9503504eefc07871